vendor/github.com/jesseduffield/gocui/gui.go

```diff
@@ -1020,10 +1020,11 @@ func (g *Gui) drawTitle(v *View, fgColor, bgColor Attribute) error {
 	}
 
 	tabs := v.Tabs
+	prefix := v.TitlePrefix
```

Comment:
We'd only have to add
```
	if prefix != "" {
		prefix += "─"
	}
```
here to make it work.

Reply:
Looks good